Update slinky version in running a validator docs (#237)
dydxprotocol/v4-documentation: pages/infrastructure_providers-validators/running_a_validator.md

Note: example endpoints will work out of the box, but will be quickly rate limited and will not be useable over long periods of blocks.

## Connect Sidecar

Starting in `v5.0.0`, running a validating full node requires a Skip Protocol's Connect Sidecar to be run in order to fetch Oracle prices. The sidecar should be started before upgrading from `v4` to `v5`. Instructions to start Connect Sidecar can be found [here](https://docs.skip.build/connect/validators/quickstart).

Support issues with Skip's Sidecar should be directed [here](https://discord.gg/7hxEThEaRQ).

> For mainnet deployment by dYdX Operation Services, Ltd. and for testnet, run the minimum required version of the Connect sidecar listed in the `dydx Blockchain` section [here](https://docs.skip.build/connect/validators/quickstart#run-connect-sidecar).

Reviewer: ```suggestion
You can find the required version of the Connect sidecar listed in the `dYdX Blockchain` section [here](https://docs.skip.build/connect/validators/quickstart#run-connect-sidecar).
```

Support issues with Skip's Sidecar should be directed [here](https://discord.gg/7hxEThEaRQ).

> For mainnet deployment by dYdX Operation Services, Ltd. run: [`v1.0.3`](https://github.com/skip-mev/slinky/tree/v1.0.3) (`ghcr.io/skip-mev/slinky-sidecar:v1.0.3`)

Reviewer 2: There is this section that Skip maintains [here](https://docs.skip.build/connect/validators/quickstart#run-connect-sidecar), that is kept up to date on which sidecar version to use for each version of dYdX (v5, v6, soon to be v7).

> For mainnet deployment by dYdX Operation Services, Ltd. run: [`v1.0.3`](https://github.com/skip-mev/slinky/tree/v1.0.3) (`ghcr.io/skip-mev/slinky-sidecar:v1.0.3`)

Author: Thanks, pushed update
